index.js

```diff
@@ -52,6 +52,11 @@ class TuyaLan {
         this.cachedAccessories = new Map();
         this.api.hap.EnergyCharacteristics = require('./lib/EnergyCharacteristics')(this.api.hap.Characteristic);
 
+        if(!this.config || !this.config.devices) {
+            this.log("No devices found. Check that you have specified them in your config.json file.");

```

Comment:
This seems like it'll just say to check the config for devices since it's not defined, will it still let the instance run despite their being nothing defined in the config? I'll try this out later today.

Reply:
Should do. I tested it on my install.

Reply:
Will install tonight, cheers.

Reply:
How'd you go?

Reply:
Sorry for the late reply, it works! I'll run it via PM2 and see if anything arrises. Thank you for the fast fix!